### DISCUSSION TITLE
Changing the ACME http-01 challenge port for testing purposes

### DISCUSSION THREAD

**Discussion by sbszcz** [smallstep/certificates#1007] [Q&A] [answered]
Hello there,

is it possible to change the ACME http-01 challenge port via some configuration properties for testing purposes? 

I am asking because there seems to be a way because the step-cli tool offers the `--http-listen` parameter.

Thanks,
Sebastian

**Comment by maraino** [comment, marked as answer]
Hi @sbszcz, the client offers `--http-listen` in case you're running it in an environment where the port 80 is restricted, but you have an ingress that proxies `:80` to let's say `:8080`. But the `step-ca` will attempt to use the port 80.

I'll suggest adding an enhancement issue in this repo asking for a debug option like this.
